src/Protocol/Email.php

```diff
@@ -571,7 +572,7 @@ private static function removeGPG(string $message): string
 	 * @param string $message Unfiltered message
 	 * @return string Message with no signature
 	 */
-	private static function removeSig(string $message): string
+	private static function removeSig(string $message): array
```

Comment:
If this method does return `array`, please double-check and update `@return` documentation, or else they are not the same.